src/setupTests.test.ts

```diff
@@ -0,0 +1,40 @@
+describe("setupTests", () => {
+  describe("assertIsDefined", () => {
+    it("handles undefined values", () => {
+      const value = undefined;
+
+      expect(() => {
+        assertIsDefined(value);
+      }).toThrow();
+    });
+
+    it("handles null values", () => {
+      const value = null;
+
+      expect(() => {
+        assertIsDefined(value);
+      }).toThrow();
+    });
+
+    it("handled not undefined values", () => {      
+      const value: number | undefined = 1 as any;
```

Comment:
```suggestion
      const value: number | undefined = 1;
```
Seems like the `as any` shouldn't be necessary.

Reply:
Without the the `as any` then typescript just types value as a number regardless of the `value: number | undefined`.

Reply:
🤔

Reply:
I added a comment about it in the code.